.appveyor.yml

```diff
@@ -40,15 +40,15 @@ init:
 
 ## Install PHP and composer, and run the appropriate composer command
 install:
-    # Upgrade to 0.10.16-beta to benefit from a bugfix for
-    # https://github.com/chocolatey/choco/issues/1843
-    - ps: choco upgrade chocolatey --pre
```

Comment:
This upgrade wasn't performed on 3.9.x, and the build worked just fine. However, it used choco 0.10.15, while the latest is 2.4.0. I moved it under the if statement and removed `--pre`.